lib/licensed/source/go.rb

```diff
@@ -140,7 +141,12 @@ def gopath
         @gopath = if path.nil? || path.empty?
                     ENV["GOPATH"]
                   else
-                    File.expand_path(path, Licensed::Git.repository_root)
+                    root = begin
```

Comment:
I could revert this change, but then the tests start failing because this code path is invoked in a context when the `pwd` is in a temporary directory which is not a git repo. This is when I made the call that I will define a fallback here.

Other places that use `repository_root` really depend on it being a git repo, but this one seems like it wants to be able to swing both ways.

Reply:
Ah gotcha 👍.  I realized this is equivalent to legacy behavior, though I'm not sure that whether the legacy behavior itself is buggy 😆.

🚢  it!